packages/web/stores/root.ts

```diff
@@ -55,6 +55,29 @@ import {
 const IS_TESTNET = process.env.NEXT_PUBLIC_IS_TESTNET === "true";
 const assets = AssetLists.flatMap((list) => list.assets);
 
+const RawAssetLists = AssetLists.map((assetList) => {
+  const assets = assetList.assets.map((asset) => {
+    const ibcTrace = asset.traces.find((trace) => trace.type === "ibc");
+    let base = asset.base;
+
+    if (ibcTrace) {
+      base = ibcTrace.counterparty.base_denom;
+    }
+
+    return {
+      ...asset,
+      base,
+    };
+  });
+
+  return {
+    ...assetList,
+    assets,
+  };
+});
```

Comment:
I believe we shouldn't modify the assetlist at the root store level. Instead, how about we move this logic into the account store?

Reply:
You could do it, in general I wouldn't do it for a generic class that tends to be controlled by a 'parent,' but in this case it's to make sure the data is in the correct format, so I would say yes

Reply:
Done